--- a/node_modules/react-native-reanimated/Common/cpp/reanimated/CSS/interpolation/transforms/TransformOperationInterpolator.cpp
+++ b/node_modules/react-native-reanimated/Common/cpp/reanimated/CSS/interpolation/transforms/TransformOperationInterpolator.cpp
@@ -141,8 +141,7 @@
 }
 
 // Specialization for resolvable operations
-template <typename TOperation>
-requires ResolvableOp<TOperation>
+template <ResolvableOp TOperation>
 TransformOperationInterpolator<TOperation>::TransformOperationInterpolator(
     const std::shared_ptr<TOperation> &defaultOperation,
     ResolvableValueInterpolatorConfig config)
@@ -150,8 +149,7 @@
     : StyleOperationInterpolator(defaultOperation), config_(std::move(config)) {}
 
-template <typename TOperation>
-requires ResolvableOp<TOperation>
+template <ResolvableOp TOperation>
 std::unique_ptr<StyleOperation> TransformOperationInterpolator<TOperation>::interpolate(
     double progress,
     const std::shared_ptr<StyleOperation> &from,
@@ -164,8 +162,7 @@
       fromOp.value.interpolate(progress, toOp.value, getResolvableValueContext(context)));
 }
 
-template <typename TOperation>
-requires ResolvableOp<TOperation>
+template <ResolvableOp TOperation>
 std::shared_ptr<StyleOperation> TransformOperationInterpolator<TOperation>::resolveOperation(
     const std::shared_ptr<StyleOperation> &operation,
     const StyleOperationsInterpolationContext &context) const {
@@ -181,8 +178,7 @@
   return std::make_shared<TOperation>(resolved.value());
 }
 
-template <typename TOperation>
-requires ResolvableOp<TOperation>
+template <ResolvableOp TOperation>
 ResolvableValueInterpolationContext TransformOperationInterpolator<TOperation>::getResolvableValueContext(
     const StyleOperationsInterpolationContext &context) const {
   return ResolvableValueInterpolationContext{
